.github/workflows/docker-push-to-registries.yml

```diff
@@ -170,7 +182,7 @@ jobs:
           # Dependabot triggered PRs don't have credentials to push images
           push: ${{ github.actor != 'dependabot[bot]' }}
           context: ${{ inputs.PATH }}
-          file: ${{ inputs.DOCKERFILE }}
+          file: ${{ inputs.PATH }}/${{ inputs.DOCKERFILE }}
```

Comment:
In ACCESS™, there is  `./Dockerfile` and in `./frontend/Dockerfile`. 

As I understand it, adding PATH was a decent way to disambiguate

Reply:
I wonder if I can instead use working-directory. 
https://telicent.atlassian.net/browse/TELFE-520

Reply:
I'm a little nervy about this, I know this has broken stuff in the past e.g. here at https://github.com/telicent-oss/smart-cache-graph/blob/main/.github/workflows/builds.yml#L44 we just pass the full path to the relevant Dockerfile in the `DOCKERFILE` input rather than relying on `PATH`

Though I guess if `PATH` defaults to `.` this shouldn't matter, it's just developers need to be aware how these two things are treated and combined together

Reply:
Robs concern Somewhat resolved in https://github.com/telicent-oss/shared-workflows/pull/45/commits/2a658c19b203ce659049b74f9cf7303e63e30854